modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -69,26 +71,35 @@ async function parseI3SNodeGeometry(arrayBuffer, tile = {}, options) {
   let vertexCount;
   let byteOffset = 0;
   let featureCount = 0;
+
   if (tile.isDracoGeometry) {
-    const decompressedGeometry = await parse(arrayBuffer, DracoLoader);
+    const decompressedGeometry = await parse(arrayBuffer, DracoLoader, {
+      parseOptions: {
+        attributeNameEntry: I3S_ATTRIBUTE_TYPE
```

Comment:
Maybe a comment here to explain what is going on?

Reply:
In compressed geometry feature-index attribute can be named in different ways. For example:  CUSTOM_ATTRIBUTES_1, CUSTOM_ATTRIBUTES_2 etc. @belom88  did special option which can detect `feature-attribute` by metadata.
In this case we always will find needed feature-index attribute.
Spec is [here](https://github.com/Esri/i3s-spec/blob/master/docs/1.7/compressedAttributes.cmn.md)